code/modules/client/preference/preferences.dm

```diff
@@ -1850,20 +1985,34 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 	character.age = age
 	character.b_type = b_type
 
-	character.change_eye_color(r_eyes, g_eyes, b_eyes)
-
 	//Head-specific
 	var/obj/item/organ/external/head/H = character.get_organ("head")
+	var/obj/item/organ/internal/eyes/eyes_organ = character.get_int_organ(/obj/item/organ/internal/eyes)
```

Comment:
This and the following 4 lines are subject to change as per the status of https://github.com/ParadiseSS13/Paradise/pull/5495